### PR TITLE
Autolathe improvements and buckle sound tweak

### PR DESCRIPTION
**Pull request by LagannPM**

### CODE REVIEW

File: code/game/machinery/autolathe.dm [hunk 1]
```diff
@@ -201,14 +201,19 @@
...
 
-
 /obj/machinery/autolathe/attackby(var/obj/item/I, var/mob/user)
 	if(default_deconstruction(I, user))
 		return
 
 	if(default_part_replacement(I, user))
 		return
 
+	if(istype(I, /obj/item/weapon/disk/autolathe_disk))
```
Comment (NanakoAC) [inline code/game/machinery/autolathe.dm]: Recycling disks should be allowed in the case that
1. The disk has/had limited uses
and
2. Its uses have been depleted to zero

File: code/game/machinery/autolathe.dm [hunk 2]
```diff
@@ -361,8 +379,14 @@
 		return FALSE
 
 	if(!eating.matter || !eating.matter.len)
-		user << SPAN_NOTICE("\The [eating] does not contain significant amounts of useful materials and cannot be accepted.")
-		return
+		user << SPAN_NOTICE("\The [src] refuses to accept \the [eating] as it has non-null license.")
```
Comment (NanakoAC) [inline code/game/machinery/autolathe.dm]: This error message makes no sense, why change it

File: code/game/machinery/autolathe.dm [hunk 3]
```diff
@@ -361,8 +379,14 @@
...
+		return FALSE
+
+	if(istype(eating, /obj/item/weapon/disk/autolathe_disk))
+		var/obj/item/weapon/disk/autolathe_disk/disk = eating
+		if(disk.license)
+			user << SPAN_NOTICE("\The [eating] does not contain significant amounts of useful materials and cannot be accepted.")
```
Comment (NanakoAC) [inline code/game/machinery/autolathe.dm]: aha now i see, you've got it swapped around

File: code/game/objects/buckling.dm [hunk 4]
```diff
@@ -81,7 +81,7 @@
 	
 	if(do_after(user, buckleTime, src))
 		if(buckle_mob(M))
-			playsound(src.loc, 'sound/machines/Custom_closetopen.ogg', 15, 1, -3)
+			playsound(src.loc, 'sound/machines/Custom_closetopen.ogg', 65, 1, -3)
```
Comment (NanakoAC) [inline code/game/objects/buckling.dm]: does this need to be so loud? what's the reasoning behind this

loud sounds are rather intrusive,

File: code/game/objects/buckling.dm [hunk 5]
```diff
@@ -106,6 +106,7 @@
 	if(do_after(user, unbuckleTime, src))
 		var/mob/living/M = unbuckle_mob()
 		if(M)
+			playsound(src.loc, 'sound/machines/Custom_closetopen.ogg', 65, 1, -3)
```
Comment (NanakoAC) [inline code/game/objects/buckling.dm]: same here, i think quieter was fine

File: code/game/machinery/autolathe.dm (hunk 1 above)
Comment (LagannPM) [reply code/game/machinery/autolathe.dm]: those cases are taken into account

File: code/game/objects/buckling.dm (hunk 4 above)
Comment (LagannPM) [reply code/game/objects/buckling.dm]: they are not, they barely noticable, before they couldnt be heard at all

File: code/game/objects/buckling.dm (hunk 5 above)
Comment (LagannPM) [reply code/game/objects/buckling.dm]: same as above

**Pull request merged**